tensorflow/contrib/cmake/tf_core_framework.cmake

```diff
@@ -233,8 +233,7 @@ if(WIN32)
   list(APPEND tf_core_lib_srcs ${tf_core_platform_windows_srcs})
 endif(WIN32)
 
-if(tensorflow_ENABLE_SSL_SUPPORT)
-  # Cloud libraries require boringssl.
```

Comment:
I think there is a problem here.
What happens if users disable GRPC support but enable GCS support?
It looks like with your proposal there is no way to add GCS support if you disable GRPC support, which look not so related.

Reply:
Thanks @gunan for the review. The GCS support requires curl, and curl is not available with CMAKE yet. I think it makes sense to disable GCS for now. I am actively looking into add curl support, will add a follow up PR once it is ready.

For now, I just disabled GCS support. Please take a look.